internal/devices/pnp.go

```diff
@@ -95,3 +95,30 @@ func readCsPipeOutput(l net.Listener, errChan chan<- error, result *[]string) {
 
 	errChan <- nil
 }
+
+// readCsPipeOutput is a helper function that connects to a listener and reads
```

Comment:
Comment needs an update

Reply:
Oh, `readCsPipeOutput` still exists above as well. Is that used anywhere anymore?

Reply:
yeah they're both used

Reply:
Ok I see it here now https://github.com/microsoft/hcsshim/blob/75d6f7115e9c4765da9ba05810d264f5e5ce5183/internal/devices/assigned_devices.go#L68

Reply:
I'll let you close this after the comment update